util/pkg/vfs/vfs.go

```diff
@@ -59,8 +59,8 @@ type Path interface {
 	// Remove deletes the file
 	Remove() error
 
-	// RemoveAll completely deletes the file (with all its versions and markers)
-	RemoveAll() error
+	// RemoveAllVersions completely deletes the file (with all its versions and markers)
```

Comment:
```suggestion
	// RemoveAllVersions completely deletes the file (with all its versions and markers).
```

Reply:
This would make it inconsistent with the other comments around it.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
I agree with the principle. Fixed.